docs/setting-up-github-actions.md

```diff
@@ -1,10 +1,10 @@
 # Setting up SBOM generation for GitHub Actions
 
-This document provides an example on how to integrate SBOM tool into GitHub Actions, you may use this as a guide to adding the tool to your GitHub action.
+Users may use the sample code provided below for integrating the SBOM tool into GitHub Actions.
 
 ## Existing setup
 
-In our Github project, the source contains a project called Sample. We also have a workflow that builds the project and saves the generated binaries as a pipeline artifact.
+In this Github project, the source contains a project called 'Sample'.  The workflow builds the project and saves the generated binaries as pipeline artifacts.
```

Comment:
nitpick: Remove extra space after the period

Reply:
> nitpick: Remove extra space after the period

Nitpicking is allowed... and (in fact) freely encouraged!! I am sure that I munged up the spacing in a few spots - ha!